lua/entities/gmod_tardis/modules/teleport/sh_tp_vortex.lua

```diff
@@ -23,7 +23,7 @@ function ENT:ToggleFastRemat()
 end
 
 function ENT:SetFastRemat(on, force)
-    if (not force) and (self:CallHook("CanToggleFastRemat") == false) then
+    if self:CallHook("CanToggleFastRemat", force) == false then
```

Comment:
These changes are not required as redecoration was fixed in a different way already

Reply:
where and how was it fixed?

Reply:
I believe it is better within the `CenToggleFastRemat` hook since even though we are restricting it for teleport/vortex, there might be other restrictions that don't even apply to force fast remat toggle

Reply:
Makes sense to me, no harm in doing this just thought worth noting